snmp/datadog_checks/snmp/snmp.py

```diff
@@ -242,7 +283,15 @@ def check(self, instance):
         """
         # Reset errors
         self._error = self._severity = None
-        config = self._config
+        if instance.get('network_address'):
+            for discovered in list(self._config.discovered_instances.values()):
+                # XXX cleanup failing hosts
```

Comment:
```suggestion
                # TODO: cleanup failing hosts
```

TODO is a common way to indicate todos, it's also well supporte by major IDEs.